src/transformers/utils/quantization_config.py

```diff
@@ -367,8 +369,9 @@ def __init__(
         module_name_preceding_first_block: Optional[List[str]] = None,
         batch_size: int = 1,
         pad_token_id: Optional[int] = None,
-        disable_exllama: bool = False,
+        disable_exllama: bool = True,
```

Comment:
is this not breaking

Reply:
Yeah, I was worried about that too. I would be great to force user to use exllamav2 instead of exllama. Otherwise, we can keep leave it to `False` and we can either: 
- set `disable_exllamav2` to `False` and we check if indeed they have access to exllamav2 kernel, otherwise the user will fallback to exllama kernel. 
- set `disable_exllamav2` to `True` and the user will have to enable it manually.
Compared to the second option, the first option will make it easy for current users and new users to discover and use exllamav2. However, it will be more verbose (fallback to exllama). 
LMK what you think =) cc @younesbelkada

Reply:
I decided to go with the second option and rename the variable